scripts/validation/providers/validate.ts

```diff
@@ -117,10 +117,23 @@ function validateProvider(providerKey: string, provider: Provider) {
     }
 
     if (provider.auth_mode === 'API_KEY') {
-        if (!provider?.credentials?.['apiKey']) {
+        if (!provider.credentials?.['apiKey']) {
             console.error(chalk.red('error'), chalk.blue(providerKey), `"credentials" > "apiKey" is not defined`);
             error = true;
         }
+        if (!provider.proxy?.verification) {
+            console.warn(chalk.yellow('warning'), chalk.blue(providerKey), `do not have "proxy" > "verification" set`);
+        }
```

Comment:
should this warning always exist regardless of the `auth_mode`?

Reply:
it's mostly there to verify credentials at save, with oauth we already know the credentials are good